v1/css.yaml

```diff
@@ -38,6 +38,7 @@ paths:
             enum:
               - base
               - pagelib
+              - pcs
```

Comment:
You are removing `pagelib` from the description, but not from the enum here. It must be one or the other :)

Reply:
Fixed - `pagelib` is deprecated but needs to remain available for app clients that still point to that endpoint. I added some clarifying comments in the documentation